lib/ddtrace/ext/runtime.rb

```diff
@@ -3,25 +3,9 @@
 module Datadog
```

Comment:
This file was partially moved, as some concepts here (span tag names) are strictly part of the tracer, and not "core".

Reply:
For these kinds of simple constants, is that distinction relevant enough? It's somewhat weird that we even have some of these repeated -- e.g. there's multiple `runtime-id`, `language`, `service`, etc across the library.

I think we're trying to abstract away too much. How often do these change and having this separation was useful? I'm guessing probably never?

Reply:
Things that are exatly the same should be the same, in my opinion. If `runtime-id` is always "the name of a Span tag that represents the runtime id" then I think we should point to the same constant, if feasible.

If they just so happen to be the same string, but are used by different products, then no.
I think you know that much 😅

I normally lean towards minimizing duplicate constants as much as possible. But that's just me.

Reply:
Fair enough. I think there's some nuance here about being the same thing or not -- I think they are, but it's a tiny thing so I'm happy to put it aside and we can always revisit later :)